doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,136 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
+set AZURE_TENANT_ID=<value>         # Windows CMD (alternatively, use export AZURE_TENANT_ID=<value> to export to the global env)
+export AZURE_TENANT_ID=<value>      # Linux shell only
+```
+*** Note: when setting these variables, do not wrap the value in quotation marks ***
 
 You are now able to log in from Python using OAuth.
 You can test with this code:
 ```python
+import os
 from azure.common.credentials import ServicePrincipalCredentials
 
 credentials = ServicePrincipalCredentials(
-    client_id =  'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL',
-    secret = 'XXXXXXXXXXXXXXXXXXXXXXXX',
-    tenant = 'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL'
+    client_id = os.environ['AZURE_CLIENT_ID'],
+    secret = os.environ['AZURE_CLIENT_SECRET'],
+    tenant = os.environ['AZURE_TENANT_ID']
 )
 ```
 
 ## Providing credentials to the tests
 
-When you run tests in playback mode,
-they use a fake credentials file,
-located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`,
-to simulate authenticating with Azure.
+When you run tests in playback mode, they use a fake credentials file, located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`, to simulate authenticating with Azure. In most scenarios you will not have to adjust this file, you will have to make edits to this file if your service uses values that are not already included in the `mgmt_settings_fake.py` file.g
 
-In live mode, you need to use actual credentials
-like those you obtained in the previous section.
-To enable the tests to use them,
-make a copy of the `mgmt_settings_fake.py` file in the same location,
-and rename it `mgmt_settings_real.py`.
+In live mode, you need to use real credentials like those you obtained in the previous section. To enable the tests to use them, make a copy of the `mgmt_settings_fake.py` file in the same location, and rename it `mgmt_settings_real.py`.
 Then make the following changes:
 
-*   Change the value of the `SUBSCRIPTION_ID` constant to your subscription ID.
-    (If you don't have it,
-    you can find it in the "Overview" section of the "Subscriptions" blade
-    in the [Azure portal](https://portal.azure.com/).)
-*   Change the `get_credentials()` function to construct and return
-    a `UserPassCredentials` or `ServicePrincipalCredentials` object
-    such as you constructed in the samples in the previous section.
-    (Don't forget to make sure the necessary imports are present as well!)
+* Change the value of the `SUBSCRIPTION_ID` constant to your subscription ID. (If you don't have it, you can find it in the "Overview" section of the "Subscriptions" blade in the [Azure portal](https://portal.azure.com/).)
+* Change the `get_credentials()` function to construct and return a `UserPassCredentials` or `ServicePrincipalCredentials` object such as you constructed in the samples in the previous section. (Don't forget to make sure the necessary imports are present as well!)
```

Comment:
This is track1 only, so I would make it in a different file even, if any, and barely "by the way if you do track1 =>"